problem_3/spec/largest_prime_factor_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'largest_prime_factor'
+
+describe PrimeFactor do
```

Comment:
I got a failing test when I set the limit to 48. I think you're algorithm is returning the largest factor of a number. I would start with some more granular tests:

```
it "returns the largest prime factor under 5"
it "returns the largest prime factor under 25"
it "returns the largest prime factor under 48"
```